frontend/src/atoms/pipelineAtom.js

```diff
@@ -51,7 +51,7 @@ export const workspaceAtom = atom({
 
 export const getPipelines = (workspace) => {
   const running = [];
-  for (const [key, pipeline] of Object.entries(workspace.executions)) {
+  for (const [pipeline] of Object.entries(workspace.executions)) {
```

Comment:
for destructuring you need to keep both pairs: `for (const [_, pipeline] of Object.entries(workspace.executions))`